7_react/react1/week1/ToDoList/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
```

Comment:
To keep the codebase clean as much as possible you should always remove unused imports. You can remove the `React` import as well as it is not required anymore in the new React versions :)

Reply:
done